src/Gaufrette/Stream/Local.php

```diff
@@ -12,9 +12,9 @@
  */
 class Local implements Stream
 {
-    private $path;
-    private $mode;
-    private $fileHandle;
+    protected $path;
+    protected $mode;
+    protected $fileHandle;
```

Comment:
So what's the fix? Copy everything over?

On Sun, Sep 20, 2015 at 8:32 AM, Christophe Coevoet <
notifications@github.com> wrote:

> In src/Gaufrette/Stream/Local.php
> <https://github.com/KnpLabs/Gaufrette/pull/334#discussion_r39927186>:
>
> > @@ -12,9 +12,9 @@
> >   */
> >  class Local implements Stream
> >  {
> > -    private $path;
> > -    private $mode;
> > -    private $fileHandle;
> > +    protected $path;
> > +    protected $mode;
> > +    protected $fileHandle;
>
> Making things protected means they become extension points which need to
> be taken into account for backward compatibility. And extension points by
> inheritance are the worse ones regarding BC layers (we implement BC layers
> in Symfony since years, and the ones involving inheritance and BC for
> protected things have always been the worse ones to implement)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnpLabs/Gaufrette/pull/334/files#r39927186>.
>